backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/interceptor/C_DocType.java

```diff
@@ -69,15 +77,19 @@ public void validateInvoicingPoolAssignment(@NonNull final I_C_DocType docType)
 				.ifPresent(docTypeInvoicingPool -> {
 					if (!docTypeInvoicingPool.getIsActive())
 					{
-						throw new AdempiereException("Invoicing Pool is not active!")
+						final ITranslatableString msg = msgBL.getTranslatableMsgText(MSG_INACTIVE_INVOICING_POOL);
+						throw new AdempiereException(msg)
```

Comment:
pls use AdempiereException(AdMessageKey), no need of msgBL

Reply:
right, thx !